LEGO1/legoworld.cpp

```diff
@@ -0,0 +1,27 @@
+#include "legoworld.h"
+
+// OFFSET: LEGO1 0x1001ca40 STUB
+LegoWorld::LegoWorld()
+{
+  // TODO
+}
+
+// OFFSET: LEGO1 0x1001dfa0 STUB
+LegoWorld::~LegoWorld()
+{
+  // TODO
+}
+
+// OFFSET: LEGO1 0x10022080 STUB
+long LegoWorld::Tickle()
+{
+  // TODO
+
+  return 0;
+}
+
+// OFFSET: LEGO1 0x1001e9d0 STUB
+void LegoWorld::Destroy()
```

Comment:
This has an argument
```suggestion
void LegoWorld::Destroy(undefined4 u)
```

Reply:
The argument is with very high probability MxBool. See 1001EE4F.